libs/forecasting/src/lib/ForecastingDashboard/Controls/TransformationCreationDialog.tsx

```diff
@@ -166,7 +154,9 @@ export class TransformationCreationDialog extends React.Component<
           <PrimaryButton
             disabled={
               transformationCombinationErrorMessage !== undefined ||
-              transformationNameErrorMessage !== undefined ||
+              this.getTransformationNameErrorMessage(
```

Comment:
nit: could getTransformationNameErrorMessage accept undefined or string and then we can move this logic into that function ?

Reply:
Good idea! I had it accept string only because it's what TextField's onGetErrorMessage expects, but the broader definition doesn't make that one fail so it works.